### PR TITLE
fix(compiler-cli): make the unused imports diagnostic easier to read 

### PR DESCRIPTION
**Pull request by crisbeto**
The unused imports diagnostic reports once on the entire initializer and then again once per unused imports. This ends up being a bit hard to follow, because in a lot of cases the code snippet looks identical.

These changes switch to highlighting the `imports:` part of the property declaration and only highlighting the unused imports without a message.

For reference, this is what the diagnostic looks like at the moment:
image

And here's what it looks like after the changes:
image

### CODE REVIEW

**Review by clydin — APPROVED** [review]

**Review by JoostK — APPROVED** [review]

**Comment by thePunderWoman** [comment]
This PR was merged into the repository by commit 3061c675392bdf58686275d1e9cc394495c15733.

The changes were merged into the following branches: main, 19.0.x
